src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchTests.java

```diff
@@ -115,6 +158,7 @@ public void mercury_search_searchNoResultsPageDisplayed() {
     Assertion.assertTrue(searchResults.isNoResultsPagePresent());
   }
 
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
   @Test(groups = "mercury_search_noSuggestionsOnSearchResultsPage")
   public void mercury_search_noSuggestionsOnSearchResultsPage() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)